package.json

```diff
@@ -14,38 +14,51 @@
     "lint": "eslint ./src/",
     "build:bundle-profile": "webpack --profile --json > stats.json",
     "bundle-profile:analyze": "yarn build:bundle-profile && webpack-bundle-analyzer ./stats.json",
-    "clean": "yarn rimraf dist"
+    "clean": "yarn rimraf dist",
+    "update-app-styles": "node scripts/consuming-app-preprocessing.js"
   },
   "devDependencies": {
     "@babel/core": "^7.1.6",
     "@babel/plugin-proposal-class-properties": "^7.1.0",
     "@babel/preset-env": "^7.1.6",
     "@babel/preset-react": "^7.0.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^23.6.0",
+    "@patternfly/patternfly-next": "file:.yalc/@patternfly/patternfly-next",
```

Comment:
This line is causing an issue for me when trying to install dependencies. 

error Package "" refers to a non-existing file '"/Users/ibolton/Development/spaxman-p-seed/patternfly-react-seed/.yalc/@patternfly/patternfly-next"'.

Reply:
Was leftovers from the heat of development. I just updated the patternfly-next version and removed the yalc references. Let me know if you see any other issues here!